pkg/cmd/roachtest/sqlalchemy.go

```diff
@@ -110,6 +110,27 @@ func runSQLAlchemy(ctx context.Context, t test.Test, c cluster.Cluster) {
 		t.Fatal(err)
 	}
 
+	// TODO(arul): This manual install of alembic should go in the
+	//  "install dependencies" above once alembic 1.7 is out.
+	if err := repeatRunE(ctx, t, c, node, "remove old sqlalchemy-alembic", `
+		sudo rm -rf /mnt/data1/sqlalchemy-alembic
+	`); err != nil {
+		t.Fatal(err)
+	}
+
+	if err := repeatGitCloneE(ctx, t, c,
+		"https://github.com/sqlalchemy/alembic", "/mnt/data1/sqlalchemy-alembic",
+		"master", node); err != nil {
```

Comment:
i feel like this should be pinned to a specific commit, otherwise if alembic changes upstream, this test could start failing suddently

Reply:
If a change gets merged into alembic/master then it has already passed all of the Alembic CI tests, including testing against the five internal SQLA dialects (mssql, mysql, oracle, postgresql, and sqlite). One could argue that it's better to test against latest master so if something does start failing we can more quickly identify if it is an issue with Alembic, or with the dialect, or the database itself. It's your call.

Reply:
yeah, we've experienced the benefits (and pains) of testing against the latest `master` version in the past, which is why the other projects are pinned to a specific version now. we found that it caused more maintenance burden than it was worth, since our nightly tests could start failing without us taking any action. we also wouldn't have time to debug right away, making it harder to debug failures that started weeks in the past. totally agree that pinning comes with the drawback of not testing against newer changes, but we've set a periodic task for our team to update the versions of all these tests.

so i vote for leaving it pinned for now.